geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/IngestRasterCommand.scala

```diff
@@ -81,7 +82,18 @@ class PathValidator extends IParameterValidator {
 
 object IngestRasterCommand {
   @Parameters(commandDescription = "Ingest a raster file or raster files in a directory into GeoMesa")
-  class IngestRasterParameters extends CreateRasterParams {
+  class IngestRasterParameters extends AccumuloConnectionParams
+    with AccumuloRasterTableParam {
+
+    @Parameter(names = Array("--write-memory"), description = "Memory allocation for ingestion operation")
```

Comment:
Nice.  It might be worth checking to see if wiring this was already in as a ticket.